examples/index.html

```diff
@@ -71,6 +72,57 @@
   <body>
     <div class="container-fluid">
 
+    <h1>Line in highchart</h1>
+    <div id="linehigh" style="height: auto;"></div>
+    <script>    
+
+      let lineDataType1 = [{marker: { fillColor: 'transparent', lineColor: 'red' },data: {"2021-02-10":2.25,"2021-02-11":2.385,"2021-02-12":2.452,"2021-02-13":2.7,"2021-02-14":2.242,"2021-02-15":2}, type : 'spline'},{marker: { fillColor: 'transparent', lineColor: 'blue' },data: {"2021-02-10":3,"2021-02-11":4,"2021-02-12":2.452,"2021-02-13":1,"2021-02-14":5,"2021-02-15":2.9}, type : 'line'} ];
+      //format true for only lineDataType2
+      let lineDataType2 = [{
+        name: "Analysis",
+        data: [{x: 0, y: 20, color:'blue'}, {x: 1, y: 50, color:'blue'}, {x: 2, y: 60, color:'blue'}, {x: 3, y: 30, color:'orange'}, {x: 4, y: 40, color:'blue'},{x: 5, y: 70, color:'blue'}],
+        type: 'line'
+      },
+      {
+        name: "Additional",
+        data: [{x: 0, y: 50, color:'orange'}, {x: 1, y: 50, color:'brown'}, {x: 2, y: 40, color:'green'}, {x: 3, y: 80, color:'purple'}, {x: 4, y: 70, color:'red'},{x: 5, y: 70, color:'green'}],
+        type: 'line'
+      }] 
+    new Chartkick.LineChart("linehigh", lineDataType2, { format: true});
+    </script>   
+
+    <h1>combined with column chart</h1>
+    <div id="Multi" style="height: auto;"></div>
+    <script>
+      let  multi_categories = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun','Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+      //format true for only lineDataType3
+      let lineDataType3 = [{marker: { fillColor: 'transparent', lineColor: 'red' },data: {"2021-02-10":2.25,"2021-02-11":2.385,"2021-02-12":2.452,"2021-02-13":2.7,"2021-02-14":2.242,"2021-02-15":2}, type : 'spline'},{marker: { fillColor: 'transparent', lineColor: 'blue' },data: {"2021-02-10":3,"2021-02-11":4,"2021-02-12":2.452,"2021-02-13":1,"2021-02-14":5,"2021-02-15":2.9}, type : 'line'} ];
+      let lineDataType4 = [{
+        name: "Analysis",
+        data: [{x: 0, y: 2.0, color:'blue'}, {x: 1, y: 5.0, color:'blue'}, {x: 2, y: 6.0, color:'blue'}, {x: 3, y: 3.0, color:'orange'}, {x: 4, y: 4.0, color:'blue'},{x: 5, y: .70, color:'blue'}],
+        type: 'line'
+      },
+      {
+        name: "Additional",
+        data: [{x: 0, y: 5.0, color:'orange'}, {x: 1, y: 5.0, color:'brown'}, {x: 2, y: 4.0, color:'green'}, {x: 3, y: .80, color:'purple'}, {x: 4, y: .70, color:'red'},{x: 5, y: .70, color:'green'}],
+        type: 'line'
+      }]
+      new Chartkick.ColumnChart("Multi", [["Data and Privacy",2.5],["Internal Security",2.385],["Organizational security",1.667],["Product Security",2.7],["Infastructure security",1.379],["Physical Security",3]], {Datas: lineDataType4, combineCharts : true, format: false});
```

Comment:
remove 'Datas' and add something meaningful for example: line_data

Reply:
ok